src/containers/ForgotPasswordForm/state/reducer.js

```diff
@@ -0,0 +1,26 @@
+import {
+  FORGOT_PASSWORD_REQUEST, FORGOT_PASSWORD_FAIL, FORGOT_PASSWORD_SUCCESS, RESEND_RESET_LINK,
+} from './types';
+
+export default (state = {}, action) => {
+  switch (action.type) {
+    case FORGOT_PASSWORD_REQUEST:
+      return {
+        ...state, error: null, message: null, isRequesting: true,
+      };
+    case FORGOT_PASSWORD_SUCCESS:
+      return {
+        ...state, ...action.payload, linkSent: true, isRequesting: false,
```

Comment:
If the number of values you are accessing goes beyond three place them in separate lines

Reply:
Done